src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -42,6 +46,15 @@ public class MainWindow extends UiPart<Stage> {
     @FXML
     private MenuItem helpMenuItem;
 
+    @FXML
+    private MenuItem linkedinMenuItem;
+
+    @FXML
+    private MenuItem canvasMenuItem;
+
+    @FXML
+    private MenuItem edurecMenuItem;
+
```

Comment:
Minor, but I think there's no need to have them here since we are not using them in this file.